lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -0,0 +1,332 @@
+  </div>
+  <div id="enroll-run-response" class="request-response"></div>
+  <div id="enroll-run-response-error" class="request-response-error"></div>
+</fieldset>
+%endif
+% if section_data['access']['admin']:
+<fieldset class="batch-enrollment membership-section">
+  <legend id="heading-batch-enrollment" class="hd hd-3">${_("Batch Enrollment")}</legend>
+  <label>
+      ${_("Enter email addresses and/or usernames separated by new lines or commas.")}
+      ${_("You will not get notification for emails that bounce, so please double-check spelling.")}
+      <textarea rows="6" name="student-ids" placeholder="${_("Email Addresses/Usernames")}" spellcheck="false"></textarea>
+  </label>
+  <div class="role">
+      <label>
+          ${_("Role of the users being enrolled.")}
+          <select name="role">
+              % for role in section_data['enrollment_role_choices']:
+                <option value="${role}">${role}</option>
+              % endfor
+          </select>
+      </label>
+  </div>
+  % if section_data['is_reason_field_enabled']:
+      <label>
+          ${_("Enter the reason why the students are to be manually enrolled or unenrolled.")}
+          ${_("This cannot be left blank and will be recorded and presented in Enrollment Reports.")}
+          ${_("Therefore, please give enough detail to account for this action.")}
+          <textarea rows="2" id="reason-field-id" name="reason-field" placeholder="${_('Reason')}" spellcheck="false"></textarea>
+      </label>
+  %endif
+  <div class="enroll-option">
+      <label class="has-hint">
+          <input type="checkbox" name="auto-enroll" id="auto-enroll" value="Auto-Enroll" checked="yes" aria-describedby="heading-batch-enrollment">
+          <span class="label-text">${_("Auto Enroll")}</span>
+          <div class="hint auto-enroll-hint">
+              <span class="hint-caret"></span>
+              <p id="auto-enroll-tip">
+                  ${Text(_("If this option is {em_start}checked{em_end}, users who have not yet registered for {platform_name} will be automatically enrolled.")).format(em_start=HTML('<em>'), em_end=HTML('</em>'), platform_name=settings.PLATFORM_NAME)}
+                  ${Text(_("If this option is left {em_start}unchecked{em_end}, users who have not yet registered for {platform_name} will not be enrolled, but will be allowed to enroll once they make an account.")).format(em_start=HTML('<em>'), em_end=HTML('</em>'), platform_name=settings.PLATFORM_NAME)}
+                  <br /><br />
+                  ${_("Checking this box has no effect if 'Unenroll' is selected.")}
+              </p>
+          </div>
+      </label>
+  </div>
+  <div class="enroll-option">
+      <label class="has-hint">
+        <input type="checkbox" name="email-students" id="email-students" value="Notify-students-by-email" aria-describedby="heading-batch-enrollment"
```

Comment:
El atributo `aria-describedby` no puede apuntar a un elemento cuyo objetivo descriptivo es de otro elemento (`legend`). Debe apuntar a un descriptor dedicado o cambiarse por `aria-description` con su descripción in situ.